src/pages/DetailsPage.js

```diff
@@ -45,6 +45,8 @@ const DetailsPage = ({personalDetails, route, translate}) => {
     // If we have a reportID param this means that we
     // arrived here via the ParticipantsPage and should be allowed to navigate back to it
     const shouldShowBackButton = Boolean(route.params.reportID);
+    const timezone = moment().tz(details.timezone.selected);
+
```

Comment:
Can we move the current time logic into a variable here please? Something like this? 

```
const timezone = moment().tz(details.timezone.selected);
const currentTime = isNaN(timezone.zoneAbbr()) ? timezone.zoneAbbr() : `${timezone.toString().split("-")[0].slice(-3)} ${timezone.zoneAbbr()}` 
```